src/habluetooth/wrappers.py

```diff
@@ -298,7 +302,8 @@ async def connect(self, **kwargs: Any) -> bool:
         address = device.address
         try:
             scanner._add_connecting(address)
-            connected = await super().connect(**kwargs)
+            await super().connect(**kwargs)
```

Comment:
The `BleakClient.connect` and `BleakClient.disconnect` methods just return `None` now. Should the Wrapper signatures be updated as well?